lib/stream/promises.js

```diff
@@ -4,7 +4,7 @@ const { ArrayPrototypePop, Promise } = require('../ours/primordials')
 const { isIterable, isNodeStream, isWebStream } = require('../internal/streams/utils')
 const { pipelineImpl: pl } = require('../internal/streams/pipeline')
 const { finished } = require('../internal/streams/end-of-stream')
-require('stream')
```

Comment:
This needs to come from the build step, not modified here.

Reply:
Ah - didn't realise that the contents of `lib/` were generated directly from the core nodejs codebase.

#520 should solve this